var/spack/repos/builtin/packages/py-pyzmq/package.py

```diff
@@ -19,19 +19,23 @@ class PyPyzmq(PythonPackage):
         'zmq.log', 'zmq.asyncio', 'zmq.devices'
     ]
 
-    version('18.1.0', sha256='32f7618b8104021bc96cbd60be4330bdf37b929e8061dbce362c9f3478a08e21')
-    version('18.0.1', sha256='7b0107992d8cc4c43d9af1c2e13d573ea761c7feb23d7e0e7da9dc963811e68f')
-    version('17.1.2', sha256='77a32350440e321466b1748e6063b34a8a73768b62cb674e7d799fbc654b7c45')
-    version('16.0.2', sha256='717dd902c3cf432b1c68e7b299ad028b0de0d0a823858e440b81d5f1baa2b1c1')
-    version('14.7.0', sha256='809a5fcc720d286c840f7f64696e60322b5b2544795a73db626f09b344d16a15')
+    version('22.3.0', sha256='8eddc033e716f8c91c6a2112f0a8ebc5e00532b4a6ae1eb0ccc48e027f9c671c')
+    version('18.1.0', sha256='93f44739db69234c013a16990e43db1aa0af3cf5a4b8b377d028ff24515fbeb3')
+    version('18.0.1', sha256='8b319805f6f7c907b101c864c3ca6cefc9db8ce0791356f180b1b644c7347e4c')
+    version('17.1.2', sha256='a72b82ac1910f2cf61a49139f4974f994984475f771b0faa730839607eeedddf')
+    version('16.0.2', sha256='0322543fff5ab6f87d11a8a099c4c07dd8a1719040084b6ce9162bcdf5c45c9d')
+    version('14.7.0', sha256='77994f80360488e7153e64e5959dc5471531d1648e3a4bff14a714d074a38cc2')
 
     depends_on('python@2.7:2.8,3.3:', type=('build', 'run'), when='@18:')
+    depends_on('python@3.6:', type=('build', 'run'), when='@22.3.0:')
     depends_on('py-cython@0.16:', type='build')
     depends_on('py-cython@0.20:', type='build', when='@18:')
-    depends_on('py-py', type=('build', 'run'))
-    depends_on('py-cffi', type=('build', 'run'))
+    depends_on('py-cython@0.29:', type='build', when='@22.3.0:')
     depends_on('py-gevent', type=('build', 'run'))
     depends_on('libzmq')
```

Comment:
```suggestion
    depends_on('libzmq@3.2:')
```
At least for the latest version